test/message/testcfg.py

```diff
@@ -101,7 +101,7 @@ def GetCommand(self):
     flags_match = FLAGS_PATTERN.search(source)
     if flags_match:
       result += flags_match.group(1).strip().split()
-    result.append(self.file)
+      result.append(self.file)
```

Comment:
I think this might be what's causing your problems. Note that the indentation does not match what is previously was. I think that can cause issues in Python.

Reply:
Oh! Nooo! 
![epic-facepalm](https://user-images.githubusercontent.com/1699357/197271957-78d0ef21-2e1c-4a7f-94f3-1d3315e8ff6b.gif)